sandbox/handlers/get_message.js

```diff
@@ -0,0 +1,65 @@
+const KSUID = require("ksuid");
+const { sendError, write_log } = require('./utils')
+
+async function get_message(req, res, next) {
+  if (req.headers["authorization"] === "banned") {
+    sendError(res, 403, "Request rejected because client service ban is in effect");
+    next();
+    return;
+  }
+
+  const messageId = req.params.messageId;
+
+
+  write_log(res, "warn", {
+    message: `/api/v1/messages/${messageId}`,
+    req: {
+      path: req.path,
+      query: req.query,
+      headers: req.rawHeaders,
+      payload: req.body
+    }
+  });
+
+  const messageReference = KSUID.randomSync(new Date()).string
+  const batchId = KSUID.randomSync(new Date()).string
+  const routingPlanId = KSUID.randomSync(new Date()).string
+
+  res.status(201).json({
```

Comment:
Could we make this JSON come from an example file on the local file system? So take the `messageId` coming in, check that we have a `{messageId}.json` file in a folder and then respond with the JSON in that file if so? If that file doesn't exist then return a 404 - this will set us up for having a set of mocked responses for specific message identifiers.

Reply:
Also the `GET` request responds with a `200` - a `201` means `CREATED`.

Reply:
I have just used a js file with an array of responses, is a file system preferable?

Reply:
Updated to use files